src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -44,8 +46,8 @@ public function __construct($key, $values = array())
             }
         }
 
-        $this->key = $key;
-        $this->values = $values;
+        $this->key = DataHelper::emptyValueCheck($key);
```

Comment:
[Suppress](https://phpmd.org/documentation/suppress-warnings.html) the Code Climate warning for using static access here, because this is a legitimate use case.

Reply:
Done